api/QCVOC.Api/Service/Controller/ServiceController.cs

```diff
@@ -0,0 +1,214 @@
+﻿// <copyright file = "ServiceController.cs" company="QC Coders (JP Dillingham, Nick Acosta, et. al.)">
+//     Copyright (c) QC Coders (JP Dillingham, Nick Acosta, et. al.). All rights reserved. Licensed under the GPLv3 license. See LICENSE file
+//     in the project root for full license information.
+// </copyright>
+
+namespace QCVOC.Api.Service.Controller
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.ModelBinding;
+    using QCVOC.Api.Common;
+    using QCVOC.Api.Common.Data.Repository;
+    using QCVOC.Api.Service.Data.DTO;
+    using QCVOC.Api.Service.Data.Model;
+    using QCVOC.Api.Security;
+
+    /// <summary>
+    ///     Provides endpoints for manipulation of Service records.
+    /// </summary>
+    [ApiVersion("1")]
+    [Route("api/v{version:apiVersion}/[controller]")]
+    [Produces("application/json")]
+    [Consumes("application/json")]
+    public class ServiceController : Controller
+    {
+
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="ServiceController"/> class.
+        /// </summary>
+        /// <param name="serviceRepository">The respository used for Service data access.</param>
+        public ServiceController(IRepository<Service> serviceRepository)
+        {
+            ServiceRepository = serviceRepository;
+        }
+
+        private IRepository<Service> ServiceRepository { get; set; }
+
+        /// <summary>
+        ///     Returns a list of Services.
+        /// </summary>
+        /// <param name="filters">Optional filtering and pagination options.</param>
+        /// <returns>See attributes.</returns>
+        /// <response code="200">The list was retrieved successfully.</response>
+        /// <response code="401">Unauthorized.</response>
+        /// <response code="500">The server encountered an error while processing the request.</response>
+        [HttpGet("")]
+        [Authorize]
+        [ProducesResponseType(typeof(IEnumerable<Service>), 200)]
+        [ProducesResponseType(401)]
+        [ProducesResponseType(typeof(Exception), 500)]
+        public IActionResult GetAll([FromQuery]ServiceFilters filters)
+        {
+            return Ok(ServiceRepository.GetAll(filters));
```

Comment:
Missing the singular `Get()`

Reply:
I wasn't sure this was necessary, but it's in now